installer/windows/build.bat

```diff
@@ -1,12 +1,17 @@
-rem set the TAG env var from the release dir
-for /f %%i in ('dir /B release') do set TAG=%%i
+@echo off
```

Comment:
why disabling echo?

Reply:
Echo prints the command that's being executed which seemed excessive. We can leave it on if you think it adds value.

Reply:
hmm not sure. I would expect at least some feedback from script to know whats going on. I don't need commands output but otherwise, I would like to see at what step of execution I am.

Reply:
We could add some more `echo` lines like in `install.bat` to get that info. My experience was that the default command echoing was not very useful.

Reply:
If its just one or two commands to execute I think we don't need any message but if a script is longer it would be nice to have some output. Also, will we see errors during execution with `@echo off`?

Reply:
Errors will be printed, command output is not suppressed, just the printing of the commands themselves.

Reply:
ok, cool :+1: